sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkEventEmitterProvider.java

```diff
@@ -100,6 +115,7 @@ private SdkEventEmitter(Logger delegateLogger, String eventDomain) {
     public void emit(String eventName, Attributes attributes) {
       delegateLogger
           .logRecordBuilder()
+          .setTimestamp(clock.now(), TimeUnit.NANOSECONDS)
```

Comment:
In contrast to the log API, the event API should automatically assign the timestamp. In part because its inconvenient for the user to specify it, but we also currently don't accept any parameters besides the event name and attribute when emitting an event.